### DISCUSSION TITLE
Desert Building

### DISCUSSION THREAD

**Discussion by ardissaps** [ejektaflex/Bountiful#196] [Mod Ideas]
The desert building should not use oak, they should use thing like smooth sandstone or sandstone itself.

**Comment by ejektaflex** [comment]
You're right. Currrently I do not have any code that substitutes in different planks depending on biome, though I'd like to do that in the future.

**Comment by ardissaps** [comment]
> You're right. Currrently I do not have any code that substitutes in different planks depending on biome, though I'd like to do that in the future.

Maybe you can take a look at this datapack? The developer of that mod change block based on village theme but only affect their new added village. The same way can be done on Bountiful side on vanilla village.
https://www.curseforge.com/minecraft/texture-packs/repurposed-structures-bountiful-datapack-compat

**Comment by ejektaflex** [comment]
This has been added in 6.0.3!
